packages/cohttp-mirage/cohttp-mirage.4.0.0/opam

```diff
@@ -22,8 +22,8 @@ depends: [
   "dune" {>= "1.1.0"}
   "mirage-flow" {>= "2.0.0"}
   "mirage-channel" {>= "4.0.0"}
-  "conduit" {>= "2.0.2" & <"3.0.0"}
-  "conduit-mirage" {>= "2.3.0" & <"3.0.0"}
+  "conduit" {>= "4.0.0"}
+  "conduit-mirage" {>= "4.0.0"}
```

Comment:
As above.
If anything we may add `"conflicts": [ "conduit" {= "3.0.0"} ]` but since it is not available, I don't think this is needed

Reply:
Let's try with another run of the CI with a422623 and if it's ok, let the solver does the job for us :+1:.